tests/scxa-config-set-no-autocreate.sh

```diff
@@ -1,11 +1,9 @@
 #!/usr/bin/env bash
```

Comment:
Any reason to move this to tests? I would expect us to use this not only on tests, at least from what I remember.

Reply:
This script is ultimately only used in the test scripts. The step is contained in the `create-scxa-$CORE-schema.sh`.